app/src/test/java/mozilla/lockbox/store/TelemetryStoreTest.kt

```diff
@@ -79,6 +81,17 @@ class TelemetryStoreTest : DisposingTest() {
         eventsObserver.assertLastValueMatches {
             it.toJSON() == action.createEvent().toJSON()
         }
+        action = RouteAction.ItemList
```

Comment:
These additions really need to be after line 95, or it could cause failures if we add other ping upload triggers.

Reply:
Adding the `ClipboardAction` checks (both "username" and "password") seems worthwhile, to verify the username or password value is NOT leaked into the event.  There's some walking of the `toJSON()` involved, but that seems straight forward to do.

Otherwise, I'm not sure there's value to the other added.